src/importer.rs

```diff
@@ -143,4 +152,49 @@ impl GtfsImporter {
         }
         Ok(())
     }
+
+    pub fn insert_stop_route_relations(
+        &self,
+        trips: &[gtfs_structures::RawTrip],
+        stop_times: &[gtfs_structures::RawStopTime],
+        stop_mapping: &std::collections::HashMap<String, String>,
+        route_mapping: &std::collections::HashMap<String, String>,
+    ) -> Result<(), anyhow::Error> {
+        for (route_gtfs_id, route_topo_id) in route_mapping {
+            for stop_gtfs_id in stops_of_route(&route_gtfs_id, trips, stop_times) {
+                let stop_topo_id = match stop_mapping.get(&stop_gtfs_id) {
+                    Some(id) => id,
+                    None => {
+                        log::warn!("Could not find wikibase id for gtfs id: {}", stop_gtfs_id);
+                        continue;
+                    }
+                };
+                let claim = api_client::claim_item(
+                    &self.query.known_entities.properties.part_of,
+                    &route_topo_id,
+                );
+
+                self.writer.client.add_claims(stop_topo_id, vec![claim])?;
+            }
+        }
+        Ok(())
+    }
+}
+
+pub fn stops_of_route(
+    route_id: &str,
+    trips: &[gtfs_structures::RawTrip],
+    stop_times: &[gtfs_structures::RawStopTime],
+) -> std::collections::HashSet<String> {
+    let mut result = std::collections::HashSet::new();
+        for trip in trips.iter().filter(|trip| trip.route_id == route_id) {
+            for stop_time in stop_times
```

Comment:
I think this quadratic loop is too expensive. You should use a `Gtfs` instead a of a `RawGtfs` and access easily the stops of a trip